### DISCUSSION TITLE
This account is blocked

### DISCUSSION THREAD

**Discussion by jmussamuhindo** [vercel/community#4951] [Account Issue]
### Question

Hi,
Could you please assist me what should i do, because when i try to create account, i got the message "This user account is blocked." and it's my first time using Vercel. 

I will be happy if you try to handle this issue.

Thank you.
![issue](https://github.com/vercel/community/assets/124691303/eb19cf23-2b2f-4e56-958b-d9a9bc97c6eb)

**Comment by KierynMo** [comment]
Same with mine. I have a support ticket that is over a month old that they have never bothered with. 
![Screenshot 2023-11-28 at 3 42 25 pm](https://github.com/vercel/community/assets/78433837/35fc2079-53e5-4831-ab7d-6997d1fb54fe)
